source/configuration_manual/authentication/user_databases_userdb.rst

```diff
@@ -97,22 +97,23 @@ Userdb settings
 
 
 .. dovecot_core:setting:: userdb_default_fields
-   :values: @string
+   :values: @strlist
    :seealso: @userdb_override_fields;dovecot_core
 
    Userdb fields (and :ref:`authentication-user_database_extra_fields`)
-   that are used, unless overwritten by the userdb driver. They are in format
-   ``key=value key2=value2 ....`` The values can contain :ref:`%variables
-   <config_variables>`. All %variables used here reflect the state BEFORE the
-   userdb lookup.
+   that are used, unless overwritten by the userdb driver. The values can
+   contain :ref:`%variables <config_variables>`. All %variables used here
```

Comment:
should %variable be quoted?

Reply:
quoted how? why?

Reply:
like \`\`%variables\`\`

Reply:
I don't think that's possible inside links?..

Reply:
there seems to be few outside

Reply:
I think better make them all links. I'll look into that.